src/ansys/fluent/core/session.py

```diff
@@ -25,11 +25,14 @@
 def parse_server_info_file(filename: str):
```

Comment:
@prmukherj when this function is called the connection has already timed out. Currently, we simply log an error, break out of the connection loop and call this function. Instead we should simply raise an exception at the point of the error (failed connection). We don't need to change this code.

Reply:
@seanpearsonuk, Thank you for the suggestion. This makes more sense. Changes applied.